R/rstar.R

```diff
@@ -0,0 +1,174 @@
+# Copyright (C) 2012, 2013, 2014, 2015, 2016, 2017, 2018 Trustees of Columbia University
```

Comment:
I am not sure this copyright is correct. Shouldn't it rather be you and Aki 2020?

Reply:
Ok, I've changed it to:

# Copyright (C) 2012, 2013, 2014, 2015, 2016, 2017, 2018 Trustees of Columbia University
# Copyright (C) 2020 Ben Lambert, Aki Vehtari

Let me know if this isn't ok, as I don't have much experience with these sorts of thing.

Reply:
done